jwt/jwt.go

```diff
@@ -0,0 +1,37 @@
+package jwt
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/golang-jwt/jwt"
+)
+
+var secretKey = []byte("farawin")
+
+func CreateToken(id string) (string, error) {
+	token := jwt.NewWithClaims(jwt.SigningMethodHS256, jwt.MapClaims{
+		"id":  id,
```

Comment:
Since our system is going to get bigger with new domains and entities, I think it's better to determine which id we are using here. (tip: use `user_id` instead of id)